zerver/openapi/zulip.yaml

```diff
@@ -3742,5 +3747,5 @@ components:
         type: array
         items:
           type: string
-        default: null
+        default: []
```

Comment:
This looks wrong; `null` means all events, while `[]` means no events.

Reply:
I think that in that case there wouldn't be a single default case but rather varying everytime. So should I remove the default tag from `event_types`?

Reply:
No; the correct default value is `null`.

Reply:
Hmm, actually, never mind, I agree with your comment -- no default is the clearest way to express this.  OK.